crates/net/eth-wire-types/src/status.rs

```diff
@@ -29,6 +29,8 @@ pub struct UnifiedStatus {
     pub earliest_block: Option<u64>,
     /// The latest block number this node has (eth/69 only).
     pub latest_block: Option<u64>,
+    /// The block range this node can serve (eth/70).
+    pub block_range: Option<BlockRange>,
```

Comment:
maybe we dont technically need this, but I think we can be exhaustive here and have all versioned fields, so let's keep this